### PR TITLE
63 fix undefined node helper

### LINKED ISSUE #63: ReferenceError: NodeHelper is not defined
Hey I've been trying to get this module working and been having a few problems. I struggled through the token generation part but got it in the end. My problem now causes the MagicMirror to remain black upon start up. The error logs are below:

```
Starting MagicMirror: v2.2.2
Loading config ...
Loading module helpers ...
No helper found for module: alert.
WARNING! Could not validate config file. Please correct syntax errors. Starting with default configuration.
Loading module helpers ...
No helper found for module: alert.
App threw an error during load
ReferenceError: NodeHelper is not defined
    at Object.module.exports.Object.defineProperty.value (/home/pi/MagicMirror/modules/MMM-awesome-alexa/node_helper.js:175:18)
    at __webpack_require__ (/home/pi/MagicMirror/modules/MMM-awesome-alexa/node_helper.js:21:30)
    at /home/pi/MagicMirror/modules/MMM-awesome-alexa/node_helper.js:64:18
    at Object.<anonymous> (/home/pi/MagicMirror/modules/MMM-awesome-alexa/node_helper.js:67:10)
    at Module._compile (module.js:556:32)
    at Object.Module._extensions..js (module.js:565:10)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
Whoops! There was an uncaught exception...
ReferenceError: NodeHelper is not defined
    at Object.module.exports.Object.defineProperty.value (/home/pi/MagicMirror/modules/MMM-awesome-alexa/node_helper.js:175:18)
    at __webpack_require__ (/home/pi/MagicMirror/modules/MMM-awesome-alexa/node_helper.js:21:30)
    at /home/pi/MagicMirror/modules/MMM-awesome-alexa/node_helper.js:64:18
    at Object.<anonymous> (/home/pi/MagicMirror/modules/MMM-awesome-alexa/node_helper.js:67:10)
    at Module._compile (module.js:556:32)
    at Object.Module._extensions..js (module.js:565:10)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
```

I'm pretty sure its not actually a syntax error with the config file as it doesn't come up with the normal syntax error screen, and I checked it using the suggest online js web checker. Here is the relevant part of the config module ( i can comment out this part and it works fine, so I know there's no issues with the rest of the config.js)

```
               {
                        module: "MMM-awesome-alexa",
                        position: "middle_center",
                        config: {
                                wakeWord: "Alexa",
                                clientId: "amzn1.application-oa2-client.*********",
                                clientSecret: "3******",
                                deviceId: "3",
                                refreshToken: "Atzr|IwEB***", //i cut off most of the token coz its super long
                                lite: false,
                                isSpeechVisualizationEnabled: true
                        }
                },
```

I have seen similar errors solved by installing missing dependencies. I tried running npm install from every directory where a package.json file exists and I haven't been able to fix the error. Any help would be greatly appreciated.

### PR DESCRIPTION
**Pull request by henrikra**
Fixes #63 

The reason for this bug is that `node_helper` was not imported. According to MagicMirror docs all `node_helper` files should have minimum setup like this:
image
And we did not have that require/import

Now it should work 👍 
Do not merge until tested ⚠️

### CODE REVIEW

**Comment by dolanmiu** [comment]
So how was this module running before this fix? (Confused) 😖 

Or is this a new bug that occurred

**Comment by henrikra** [comment]
Lets see when @I-Acinonyx tests this out and see if it worked or not :D

**Comment by henrikra** [comment]
It worked! https://github.com/dolanmiu/MMM-awesome-alexa/issues/63#issuecomment-357583440

**Comment by dolanmiu** [comment]
I see there is a new `@types` folder, could you move the `my-typings` folder into it?

It serves a similar purpose!

File: tsconfig.json [hunk 1]
```diff
@@ -6,7 +6,10 @@
         "sourceMap": true,
         "declaration": false,
         "rootDir": "./src",
-        "noUnusedLocals": true
-        // "strictNullChecks": true
+        "noUnusedLocals": true,
+        "typeRoots": [
+            "@types",
```
Comment (dolanmiu) [inline tsconfig.json]: I like this

**Pull request merged**